README.md

```diff
@@ -20,11 +20,11 @@ do](https://18f.gsa.gov/2017/01/04/tracking-the-us-governments-progress-on-movin
 
 `pshtt` is a collaboration between the Department of Homeland
```

Comment:
I think that in most instances we can replace "Department of Homeland Security" with CISA (expanded or no). Yes, CISA is part of DHS, but no one says DHS TSA or DHS CBP. We're just CISA.

Reply:
Fixed in 3f30e07.